README.md

```diff
@@ -62,6 +62,38 @@ $ make clean
 
 will remove build artifacts.
 
+### Frontend
```

Comment:
I think if we are going to have a "Frontend" heading for the frontend part of the README, we should equivalently have a "Backend" heading for the backend part, otherwise it may be inconsistent and less clear. So it would be good to update the existing text to add that.

Reply:
Thanks for the great feedback! I was actually going to open an issue for fixing the formatting on the backend set up section, I think it's a little confusing, but I'll fix it in this PR if that works.

Reply:
Yup, go for it!